### DISCUSSION TITLE
Is it supported for Win 7?

### DISCUSSION THREAD

**Discussion by harryray33** [setvisible/DownZemAll#172] [Q&A]
Does this work with Win 7?

Thanks

**Comment by therube** [comment]
Win7, SP1, x-64
Appears not.
Here is the error message:

---------------------------
DownZemAll.exe - Entry Point Not Found
---------------------------
The procedure entry point CreateEventW could not be located in the dynamic link library api-ms-win-core-synch-l1-2-0.dll. 
---------------------------
OK   
---------------------------


Most likely the issue is, https://github.com/setvisible/DownZemAll/issues/99.

Seemingly https://github.com/setvisible/DownZemAll/releases/tag/v2.5.8 should work.
(Final release before the change to QT6.)

**Comment by setvisible** [comment]
Hi,
indeed Qt6 doesn't support Windows 7:
https://doc.qt.io/qt-6/supported-platforms.html

For me it's too much work to maintain Qt6 and Qt5 together (...Qt5 is very different from Qt6), so I choosed to abandon the Qt5 compatibility. However backward compatibility (in Qt6) is maintained. It permits to build the application whateever the Qt6.x version is (ex: anymone can build it with Qt6.3, Qt6.4, Qt6.5 etc.).

At this point your options are to fork from v2.5.8

Regards

**Comment by adeii** [comment]
But Qt could be ported to Win 7 and then it works>
https://github.com/adeii/Qt6Win7/releases/download/Test652/ArrowDL-v4.0.2-Qt652-x64-Win7.7z
